<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="BLU-LP-00004" name="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner" tests="9" failures="1" timestamp="2020-01-21T21:25:47 SGT" time="670.879" errors="0">
  <testcase name="tc07_editGender" time="53.193" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc05_editEmail" time="88.044" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc02_UnableToEditFirstName" time="79.219" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc07_editMobileAlreadyExists" time="60.610" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner">
    <failure type="java.lang.AssertionError" message="expected [94561718] but found [+65 94561718]">
      <![CDATA[java.lang.AssertionError: expected [94561718] but found [+65 94561718]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner.tc07_editMobileAlreadyExists(BluMyDetailsTestRunner.java:190)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- tc07_editMobileAlreadyExists -->
  <testcase name="tc06_editEmailAlreadyExists" time="75.427" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc01_editFirstName" time="83.262" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc04_unableToEditLastName" time="89.888" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc03_editLastName" time="75.593" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
  <testcase name="tc08_referAFriend" time="65.643" classname="bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner"/>
</testsuite> <!-- bluAndroid.bluAndroid.TestScripts.BluMyDetailsTestRunner -->
